ml-agents/mlagents/trainers/bc/models.py

```diff
@@ -15,7 +15,9 @@ def __init__(
         use_recurrent=False,
         seed=0,
     ):
-        LearningModel.__init__(self, m_size, normalize, use_recurrent, brain, seed)
+        LearningModel.__init__(
```

Comment:
The pythonic way to call the parent class initializer is `super().__init__` https://stackoverflow.com/a/12557639/224264

Reply:
Hmm - this code shouldn't be part of this PR, actually. Let me clean this up...

As far as the RewardSignals, I've changed them to use the super() call as appropriate.